cesql/spec.md

```diff
@@ -56,7 +56,10 @@ producer, or in an intermediary, and it can be implemented using any technology
 The CloudEvents Expression Language assumes the input always includes, but is not limited to, a single valid and
 type-checked CloudEvent instance. An expression MUST NOT mutate the value of the input CloudEvent instance, nor any of
 the other input values. The evaluation of an expression observes the concept of [referential
-transparency][referential-transparency-wiki]. The output of a CESQL expression evaluation is always a _boolean_, an _integer_ or a _string_, and it might include an error.
+transparency][referential-transparency-wiki]. The primary output of a CESQL expression evaluation is always a _boolean_, an _integer_ or a _string_.
+The secondary output of a CESQL expression evaluation is a set of errors which occurred during evaluation. This set MAY be empty, indicating that no
+error occurred during execution of the expression. The value used by CESQL engines to represent an empty set of errors is out of the scope of this
```

Comment:
Isn't the entire representation of, or even access to, errors out of scope of the spec, not just the empty set of errors?